fvm/src/executor/default.rs

```diff
@@ -79,8 +79,11 @@ where
 
                 Ok(ret)
             });
-            let (gas_used, backtrace, machine) = cm.finish();
-            (Ok((result, gas_used, backtrace)), machine)
+            let (res, machine) = cm.finish();
+            (
+                Ok((result, res.gas_used, res.backtrace, res.exec_trace)),
```

Comment:
```suggestion
                Ok(((result, res), machine)),
```

?

Reply:
Eh, but we use `gas_used` so many times, and they're not suuuper related concepts, and I don't like it.

Reply:
We can destructure this on the outside. I'm just suggesting a way to slightly reduce the amount of code.

Reply:
(but it doesn't really matter)